### ISSUE TITLE
Error: 'Namespace' object has no attribute 'tag_override'

### ISSUE THREAD

**Issue by RudStar** [jrnewell/spotify-ripper#291]
Hallo, here i get an error with v2.10.6 on Python 2.7.12
v2.10.5 is working!

Thank you!

```
Spotify error detected
'Namespace' object has no attribute 'tag_override'
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/spotify_ripper/ripper.py", line 337, in run
    set_metadata_tags(args, self.audio_file, idx, track, self)
  File "/usr/local/lib/python2.7/site-packages/spotify_ripper/tags.py", line 497, in set_metadata_tags
    tags = Id3Tags(args, audio_file, idx, track, ripper)
  File "/usr/local/lib/python2.7/site-packages/spotify_ripper/tags.py", line 195, in __init__
    super(Id3Tags, self).__init__(args, audio_file, idx, track, ripper)
  File "/usr/local/lib/python2.7/site-packages/spotify_ripper/tags.py", line 26, in __init__
    self.override_tags(idx, track, ripper)
  File "/usr/local/lib/python2.7/site-packages/spotify_ripper/tags.py", line 85, in override_tags
    tag_overrides = args.tag_override if args.tag_override is not None else []
AttributeError: 'Namespace' object has no attribute 'tag_override'
Skipping to next track...
Deleting partially ripped file
```

**Comment by jrnewell** [comment]
That is weird because you are using code that was head (non-released) but it says you are using 2.10.6 version.  That code should be for 2.10.7 version, can you upgrade and retry?

**Comment by RudStar** [comment]
Yes, it's working. Thank you!
